tweets/models.py

```diff
@@ -64,3 +66,42 @@ def like_set(self):
             content_type=content_type,
             object_id=self.id,
         ).order_by('-created_at')
+
+
+class TweetPhoto(models.Model):
+    tweet = models.ForeignKey(Tweet, on_delete=models.SET_NULL, null=True)
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
```

Comment:
user field could be convenient here because lots of cases might involved tweet.user, so this way could avoid JOIN query.
However, it seems that at DB level, it doesn't have a check to ensure that user == tweet.user.
You might need to consider this check at API level.

Reply:
Definitely!